src/components/TodoItem/components/createToggleButton.js

```diff
@@ -0,0 +1,16 @@
+import './toggleButton.scss';
+
+function createToggleButton(status = false) {
```

Comment:
Do you think `createCheckButton` would be a better function name?

Reply:
> I think you could add more test cases for all the components.

Nice! I will consider it in the next PR's